config/params.php

```diff
@@ -25,6 +35,20 @@
         RequestCollector::class,
         RouterCollector::class,
         MiddlewareCollector::class,
+        ServiceCollector::class
+    ],
+    'debugger.trackedServices' => [
```

Comment:
Maybe it's better to use snake case everywhere by analogy with "event_handlers"

Reply:
Maybe it is better to use camelCase for 'event_handlers' because we use it in other places/packages?

Reply:
Yes you're right

Reply:
Done